tutorials/spectral-cube-reprojection/SpectralCubeReprojectExample.ipynb

```diff
@@ -0,0 +1,640 @@
+{
```

Comment:
<ul><li>keyword: spectral cube, radio astronomy, astroquery, units, dask</li><li>Summary: Also use astroquery and splatalogue to identify the emission lines present in the data. Something about dask.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/504/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>added</p>